include/swift/AST/LocalizationLanguages.def

```diff
@@ -10,7 +10,7 @@
 //
 //===----------------------------------------------------------------------===//
 //
-//  This file has all languages currently supported in localization.
+//  This file has all languages for localization.
```

Comment:
This file now has all of ISO 639-1 codes and languages. I got them from this [GitHub Gist](https://gist.github.com/DimazzzZ/4e2a5a6c8c6f67900091#gistcomment-3272769).

Reply:
Sounds good!

Reply:
@HassanElDesouky @xedin This is just nit comment, but do you think will be better for this comment to mention the ISO e.g. `This file has all language codes defined by ISO 639-1  for localization` ... because before I read this that was my first question... if "all languages" is coming from some standard, so maybe it is worth a mention?

Reply:
Yes, it's worth mentioning. I'll add it now.